data/autoyast_opensuse/rule-based_example/classes/general/users.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
also please do squash with the corresponding commit, so all the changes to xml will be in the same commit. We don't need to remark the differences between openSUSE and SLE via commits, if that was the case we would prefer to share code between them, so I guess the best is having in one commit.

Reply:
Did all the editiing and rebased it so that it is just one commit.